configs/machines/blogin.yaml

```diff
@@ -47,11 +47,11 @@ pool_dir:  "/scratch/usr/hbkawi"
 
 # default settings for compiler, mpi and I/O libs
 # TODO: system_libs not yet properly configured as I (seb-wahl) don't use them
-compiler_mpi: intel2019_impi2019
+#compiler_mpi: intel2019_impi2019
 #iolibraries: system_libs
 #
 # for FOCIOIFS use
-# compiler_mpi: intel2019_ompi
+compiler_mpi: intel2019_ompi
```

Comment:
The default should not be changed here. This can be done in 2 ways:
1) Include this in the computer section of `awicm3.yaml` inside a `choose_computer.name`
2) select it from the runscript `computer.compiler_mpi: intel2019_ompi` and from `esm_master install-awicm3... -m open_mpi.yaml` where `open_mpi.yaml` is:
```
computer:
    compiler_mpi: intel2019_ompi
```